test/libsolidity/SemanticTest.cpp

```diff
@@ -164,6 +165,36 @@ map<string, Builtin> SemanticTest::makeBuiltins()
 				return toBigEndian(h256(ExecutionFramework::setAccount(accountNumber).asBytes(), h256::AlignRight));
 			}
 		},
+		{
+			"bytecode",
+			[this](FunctionCall const& _call) -> optional<bytes>
+			{
+				bool result{false};
+				optional<regex> regexPattern;
+				soltestAssert(_call.arguments.parameters.size() == 1, "Bytecode pattern string expected.");
+				string pattern{_call.arguments.parameters.at(0).rawString};
+				bytes bytecode = ExecutionFramework::bytecode();
+				if (pattern.length() >= 2 && boost::starts_with(pattern, "\"") && boost::ends_with(pattern, "\""))
+				{
+					pattern = pattern.substr(1, pattern.length() - 2);
+					try
+					{
+						regex expr(pattern);
+						regexPattern = expr;
+					}
+					catch(...)
```

Comment:
What exception types are you expecting here? I think you should be catching and silencing only that particular exception and not everything.